StringFormatter/Numeric.cs

```diff
@@ -216,6 +217,7 @@ unsafe static partial class Numeric {
                 NumberToCustomFormatString(formatter, ref number, specifier, culture);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
I'm a little leery of marking these as aggressive inline; they're not especially small. Does this attribute make the benchmark results better?